### PR TITLE
(SIMP-8398) Set safe puppetserver default ciphers

### PR DESCRIPTION
**Pull request by trevor-vaughan**
SIMP-8398 #close

### CODE REVIEW

File: manifests/master.pp [hunk 1]
```diff
@@ -138,6 +138,8 @@
 #   list is Java version dependent and you will need to check the system Java
 #   documentaiton for details.
 #
+#   * The default is in the module data
```
Comment (lnemsick-simp) [inline manifests/master.pp:141]: Line 139 above has a typo: documentaiton -> documentation.  Would you please fix and regenerate the REFERENCE.md?

**Comment by trevor-vaughan** [comment]
Unsure why it didn't hook together but GitLab is [OK](https://gitlab.com/simp/pupmod-simp-pupmod/-/pipelines/193557317)

**Comment by lnemsick-simp** [comment]
It did show the pipeline before the 'fix typo' commit.

**Comment by trevor-vaughan** [comment]
> 
> 
> It did show the pipeline before the 'fix typo' commit.

Oh, well a new push would be no different than Travis.

**Review by op-ct — APPROVED** [review]
👁️👍 

The `.gitlab-ci.yml` changes are a good idea; I'll add them to the next puppetsync.

**Pull request merged**